.circleci/config.yml

```diff
@@ -13,23 +13,20 @@ jobs:
       - checkout
       - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS $XCODE_TEST_REPORTS
       - restore_cache:
-          keys:
-            - v1-dep-{{ .Branch }}-
-            - v1-dep-master-
-            - v1-dep-
-      - run: 
-          name: Bootstrap Carthage
-          command: scripts/bootstrap-if-needed.sh
+          key: RxAlamofire-Test-Carthage-Dependency
+      - run:
+          name: Install Carthage
+          command: set -o pipefail && carthage update --cache-builds --new-resolver --configuration Debug
```

Comment:
Very interesting! I didn't know `--cache-builds` and `--new-resolver` are a thing. Mind sharing a link what they do ? Out of curiosity mainly.